core/src/HIP/Kokkos_HIP_Instance.cpp

```diff
@@ -97,10 +97,14 @@ void HIPInternal::print_configuration(std::ostream &s) const {
     KOKKOS_IMPL_HIP_SAFE_CALL(hipGetDeviceProperties(&hipProp, i));
 
     s << "Kokkos::HIP[ " << i << " ] "
-      << "gcnArch " << hipProp.gcnArch << ", Total Global Memory: "
+      << "gcnArch " << hipProp.gcnArchName << ", Total Global Memory: "
       << ::Kokkos::Impl::human_memory_size(hipProp.totalGlobalMem)
       << ", Shared Memory per Block: "
-      << ::Kokkos::Impl::human_memory_size(hipProp.sharedMemPerBlock);
+      << ::Kokkos::Impl::human_memory_size(hipProp.sharedMemPerBlock)
+      << ", APU: " << hipProp.integrated
+      << ", Is Large Bar: " << hipProp.isLargeBar
```

Comment:
What is that one useful for?
I don't have an issue with it I just don't get what it tells us.

Reply:
When it's enable the GPU can access regular host memory. A couple of people on slack have been using this capability but it's not portable across all AMD GPUs